packages/cyberstorm/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./Tooltip.module.css";
+
+export interface TooltipProps {
+  content?: ReactNode;
+  sideOffset?: number;
+  side?: "right" | "top" | "bottom" | "left" | undefined;
+  collisionPadding?: number;
+  sticky?: "always" | "partial" | undefined;
+  avoidCollisions?: boolean;
+}
+
+// Cyberstorm Tooltip component
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
+  content,
+  sideOffset = 5,
+  side = "right",
+  collisionPadding = 20,
+  sticky = "always",
+  avoidCollisions = true,
```

Comment:
I'd remove these default values from here and move them to a Tooltip.defaultProps (same as in Button.tsx for example).

Reply:
I'd leave these here, since we probably want to have the same expected behavior of the Tooltip everywhere.

Reply:
They'd still exist, just in a different place, the functionality would be the same, but this would be in line with all of the other components. Those values can be overridden as well as with the other style. 🤔 Is there some other reason I'm not seeing? All in all just a small little thing, should not become a blocker. @Cyanideankst do whatever feels right 🤷

Reply:
Got an answer through discord. Does not require any modifications.